packages/babel-plugin-ember-test-metadata/utils.js

```diff
@@ -56,48 +56,73 @@ function getProjectConfiguration(project) {
   return parsedProjectConfiguration;
 }
 
-function _getParsedClassicFilepath(pathSegments, projectConfiguration) {
-  const projectNamePathSeparators = projectConfiguration.pkg.name.split(path.sep);
+function _getClassicPath(preformattedPath, projectName) {
+  const formattedPath = preformattedPath
+    .slice(preformattedPath.lastIndexOf(projectName), preformattedPath.length)
+    .replace(`${projectName}/`, '');
 
-  pathSegments.splice(
-    0,
-    pathSegments.indexOf(projectNamePathSeparators[0]) + projectNamePathSeparators.length
-  );
-
-  return pathSegments.join(path.sep);
+  return formattedPath;
```

Comment:
We should seriously consider not using arrays for path manipulation. I made this comment before, and I believe Glenn made those changes to use `path.parse` instead. Using arrays is error prone when you're working with structured strings, so we should do our best to avoid things that could get us into trouble in the future.

We don't have to do those changes in this PR.